src/nacl/hash.py

```diff
@@ -11,67 +11,106 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+"""
+The :mod:`nacl.hash` module exposes one-shot interfaces
+for libsodium selected hash primitives and the constants needed
+for their usage.
+"""
 
 from __future__ import absolute_import, division, print_function
 
 import nacl.bindings
 import nacl.encoding
 
+
 BLAKE2B_BYTES = nacl.bindings.crypto_generichash_BYTES
+"""Default digest size for :func:`blake2b` hash"""
```

Comment:
I never use docstrings like this so you'll have to teach me: they go after the line?

Reply:
Yes, they must go after the line they refer to